PatientWebApplication/PatientWebApplicationIntegrationTests/DoctorAppointmentControllerSetup.cs

```diff
@@ -0,0 +1,88 @@
+﻿using HealthClinic.CL.DbContextModel;
+using HealthClinic.CL.Model.Doctor;
+using HealthClinic.CL.Model.Employee;
+using HealthClinic.CL.Model.Patient;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PatientWebApplicationIntegrationTests
+{
+    public static class DoctorAppointmentControllerSetup
+    {
+        public static MyDbContext DbContext { get; set; }
+
+        public static MyDbContext SetupDbContext()
+        {
+            DoctorUser doctor1 = new DoctorUser(1, "TestDoctorName1", "TestDoctorNameSurname1", "1234", "02/02/2020", "123", "email", "pass", "Grad", 200.0, false, "Cardiology", new List<DoctorNotification>(), "Ordination 2");
+            DoctorUser doctor2 = new DoctorUser(2, "TestDoctorName2", "TestDoctorNameSurname2", "1234", "02/02/2020", "123", "email", "pass", "Grad", 200.0, false, "Pulmonology", new List<DoctorNotification>(), "Ordination 2");
+            DoctorUser doctor3 = new DoctorUser(3, "TestDoctorName3", "TestDoctorNameSurname3", "1234", "02/02/2020", "123", "email", "pass", "Grad", 200.0, false, "Cardiology", new List<DoctorNotification>(), "Ordination 2");
+
+            DbContext.Doctors.Add(doctor1);
+            DbContext.Doctors.Add(doctor2);
+            DbContext.Doctors.Add(doctor3);
+
+            PatientUser patient1 = new PatientUser(1, "PatientName1", "PatientSurname1", "Female", "1234", "2/2/2020", "123", "2112313", "Alergija", "Grad", false, "email", "pass", false, "Grad2", "Roditelj", null);
+            PatientUser patient2 = new PatientUser(2, "PatientName2", "PatientSurname2", "Female", "1234", "2/2/2020", "123", "2112313", "Alergija", "Grad", false, "email", "pass", false, "Grad2", "Roditelj", null);
+            DbContext.Patients.Add(patient1);
+            DbContext.Patients.Add(patient2);
```

Comment:
Forward patient objects directly as a parameter to the Add () method. For example:
"DbContext.Patients.Add(new PatientUser(1, "PatientName1", "PatientSurname1", "Female", "1234", "2/2/2020", "123", "2112313", "Alergija", "Grad", false, "email", "pass", false, "Grad2", "Roditelj", null));"